lib/appsignal/cli/diagnose.rb

```diff
@@ -285,6 +285,7 @@ def library_information
             save :language, "ruby"
             puts_and_save :package_version, "Gem version", Appsignal::VERSION
             puts_and_save :agent_version, "Agent version", Appsignal::Extension.agent_version
+            puts_and_save :agent_platform, "Agent platform", Appsignal::System.agent_platform
```

Comment:
Over the weekend I was wondering if this was a good approach, trying to figure out the platform during the diagnose separately. If people use `APPSIGNAL_BUILD_FOR_MUSL` during installation, and the env var is not a permanent part of their system environment, we wouldn't know for sure which build they've actually installed during `appsignal diagnose`.

Would it be an idea to save the build platform to a file the `ext` dir during installation and read it from that `.appsignal_platform` file when the diagnose is run? That way we'd be sure that the diagnose reports the installed version, rather than the detected version on `appsignal diagnose` runs.

WDTY @thijsc @jeffkreeftmeijer ?